cfg/17.06/definitions.yaml

```diff
@@ -453,12 +453,15 @@ groups:
     description: "Ensure Userland Proxy is Disabled (Scored)"
     audit: ps -ef | grep dockerd
     tests:
+      bin_op: or
       test_items:
       - flag: "--userland-proxy"
         compare:
           op: eq
           value: "false"
         set: true
+      - flag: "--userland-proxy"
```

Comment:
If the --userland-proxy flag isn't present, the default will apply, so it will be enabled, right?  I think the check simply needs to ensure that --userland-proxy=false (matching what the CIS spec says)

Reply:
@lizrice , I can do this, but in case the --userland-proxy is missing in the docker cmd line because for example someone edited the docker config file and removed the --userland-proxy, the check will fail with false negative. I tried to cover this case. If this is redundant, I will remove the check, but then we assume that --userland-proxy is always present with true or false value.

Reply:
Yes, from the spec I believe this check is meant to fail when --userland-proxy is not present (because it will be falling back to the default value of enabled)